tools/diar_eval.py

```
@@ -0,0 +1,511 @@
+            raise SystemExit(f"Lauf '{name}' gibt es nicht ({pfad}). Vorhanden: "
+                             + (", ".join(sorted(
+                                 f[:-5] for f in os.listdir(os.path.join(EVAL, "laeufe"))
+                                 if f.endswith(".json")))
+                                if os.path.isdir(os.path.join(EVAL, "laeufe")) else "keine"))
+        with open(pfad, encoding="utf-8") as f:
+            lauf = json.load(f)
+        # Form pruefen statt sich auf sie zu verlassen: die Laufdateien liegen unter eval/ und
+        # koennen von Hand entstanden oder halb geschrieben sein (der Lauf schreibt nach JEDER
+        # Datei). Ohne die Wache endet das in einem KeyError-Traceback mitten in der Tabelle.
+        if not isinstance(lauf, dict) or not isinstance(lauf.get("dateien"), dict):
+            raise SystemExit(f"'{name}' ist keine Laufdatei ({pfad}).")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)